File: src/tool/spring-boot-generator/output/test/test/src/main/java/com/test/pojo/entity/LineInfo.java
package com.test.pojo.entity;

import com.test.pojo.entity.LineInfo;
import io.swagger.annotations.ApiModel;
import io.swagger.annotations.ApiModelProperty;
import lombok.Getter;
import lombok.Setter;

import java.util.Date;
import javax.validation.constraints.NotNull;



@ApiModel("线路信息")
@Getter
@Setter
public class LineInfo @extends {

    @ApiModelProperty("道床类别")
    private Integer ballastType;

    @ApiModelProperty("投产日期")
    private Integer commissioningDate;

    @ApiModelProperty("创建时间")
    private Date createTime;

    @ApiModelProperty("线路里程")
    private Float distance;

    @ApiModelProperty("线路全称")
    @NotNull
    private String fullName;

    @ApiModelProperty("轨距类型")
    @NotNull
    private Integer gaugeType;

    @ApiModelProperty("线路等级")
    @NotNull
    private Integer grade;

    @ApiModelProperty("线路编号")
    @NotNull
    private Integer id;

    @ApiModelProperty("输入代码")
    @NotNull
@NotNull
    private String inputCode;

    @ApiModelProperty("删除标记")
    @NotNull
@NotNull
@NotNull
    private Integer isDelete;

    @ApiModelProperty("限制坡度")
    @NotNull
@NotNull
@NotNull
    private Float limitGrad;

    @ApiModelProperty("线路干支")
    @NotNull
@NotNull
@NotNull
    private Integer lineBranches;

    @ApiModelProperty("线路编码")
    @NotNull
@NotNull
@NotNull
@NotNull
    private String lineCode;

    @ApiModelProperty("正线数目")
    @NotNull
@NotNull
@NotNull
@NotNull
    private Integer lineNum;

    @ApiModelProperty("最小曲线半径")
    @NotNull
@NotNull
@NotNull
@NotNull
    private Float minRadiusCurve;

    @ApiModelProperty("线路性质")
    @NotNull
@NotNull
@NotNull
@NotNull
    private Integer nature;

    @ApiModelProperty("运营类型")
    @NotNull
@NotNull
@NotNull
@NotNull
    private Integer operateType;

    @ApiModelProperty("操作人")
    @NotNull
@NotNull
@NotNull
@NotNull
    private Integer operator;

    @ApiModelProperty("线路简称")
    @NotNull
@NotNull
@NotNull
@NotNull
@NotNull
    private String shortName;

    @ApiModelProperty("状态")
    @NotNull
@NotNull
@NotNull
@NotNull
@NotNull
    private Integer status;

    @ApiModelProperty("轨道类型")
    @NotNull
@NotNull
@NotNull
@NotNull
@NotNull
    private Integer trackType;

    @ApiModelProperty("牵引类型")
    @NotNull
@NotNull
@NotNull
@NotNull
@NotNull
    private Integer tractionType;

    @ApiModelProperty("更新时间")
    @NotNull
@NotNull
@NotNull
@NotNull
@NotNull
    private Date updateTime;

    @ApiModelProperty("年运量(万吨)")
    @NotNull
@NotNull
@NotNull
@NotNull
@NotNull
    private Float yearVolume;

}

File: src/tool/spring-boot-generator/output/railway/db/lineInfo.js
let lineInfo = {
	name: "lineInfo",
	description: "线路信息",
	columns: {
		ballastType: {
			name: "ballastType",
			description: "道床类别",
			nullable: true,
			type: {
				name: "int",
				length: null
			}
		},
		commissioningDate: {
			name: "commissioningDate",
			description: "投产日期",
			nullable: true,
			type: {
				name: "date"
			}
		},
		createTime: {
			name: "createTime",
			description: "创建时间",
			nullable: true,
			type: {
				name: "datetime"
			}
		},
		distance: {
			name: "distance",
			description: "线路里程",
			nullable: true,
			type: {
				name: "float"
			}
		},
		fullName: {
			name: "fullName",
			description: "线路全称",
			nullable: false,
			type: {
				name: "varchar",
				length: null
			}
		},
		gaugeType: {
			name: "gaugeType",
			description: "轨距类型",
			nullable: true,
			type: {
				name: "int",
				length: null
			}
		},
		grade: {
			name: "grade",
			description: "线路等级",
			nullable: true,
			type: {
				name: "int",
				length: null
			}
		},
		id: {
			name: "id",
			description: "线路编号",
			isPk: true,
			autoIncrement: true,
			nullable: false,
			type: {
				name: "int",
				length: null
			}
		},
		inputCode: {
			name: "inputCode",
			description: "输入代码",
			nullable: false,
			type: {
				name: "varchar",
				length: null
			}
		},
		isDelete: {
			name: "isDelete",
			description: "删除标记",
			nullable: false,
			type: {
				name: "int",
				length: null
			}
		},
		limitGrad: {
			name: "limitGrad",
			description: "限制坡度",
			nullable: true,
			type: {
				name: "float"
			}
		},
		lineBranches: {
			name: "lineBranches",
			description: "线路干支",
			nullable: true,
			type: {
				name: "int",
				length: null
			}
		},
		lineCode: {
			name: "lineCode",
			description: "线路编码",
			nullable: false,
			type: {
				name: "varchar",
				length: null
			}
		},
		lineNum: {
			name: "lineNum",
			description: "正线数目",
			nullable: true,
			type: {
				name: "int",
				length: null
			}
		},
		minRadiusCurve: {
			name: "minRadiusCurve",
			description: "最小曲线半径",
			nullable: true,
			type: {
				name: "float"
			}
		},
		nature: {
			name: "nature",
			description: "线路性质",
			nullable: true,
			type: {
				name: "int",
				length: null
			}
		},
		operateType: {
			name: "operateType",
			description: "运营类型",
			nullable: true,
			type: {
				name: "int",
				length: null
			}
		},
		operator: {
			name: "operator",
			description: "操作人",
			nullable: true,
			type: {
				name: "int",
				length: null
			}
		},
		shortName: {
			name: "shortName",
			description: "线路简称",
			nullable: false,
			type: {
				name: "varchar",
				length: null
			}
		},
		status: {
			name: "status",
			description: "状态",
			nullable: true,
			type: {
				name: "int",
				length: null
			}
		},
		trackType: {
			name: "trackType",
			description: "轨道类型",
			nullable: true,
			type: {
				name: "int",
				length: null
			}
		},
		tractionType: {
			name: "tractionType",
			description: "牵引类型",
			nullable: true,
			type: {
				name: "int",
				length: null
			}
		},
		updateTime: {
			name: "updateTime",
			description: "更新时间",
			nullable: true,
			type: {
				name: "datetime"
			}
		},
		yearVolume: {
			name: "yearVolume",
			description: "年运量(万吨)",
			nullable: true,
			type: {
				name: "float"
			}
		}
	}
};

let columnsArray = [
	"ballastType",
	"commissioningDate",
	"createTime",
	"distance",
	"fullName",
	"gaugeType",
	"grade",
	"id",
	"inputCode",
	"isDelete",
	"limitGrad",
	"lineBranches",
	"lineCode",
	"lineNum",
	"minRadiusCurve",
	"nature",
	"operateType",
	"operator",
	"shortName",
	"status",
	"trackType",
	"tractionType",
	"updateTime",
	"yearVolume"
];

function create() {
  return {};
}

lineInfo.columnsArray=columnsArray;
lineInfo.create=create;
module.exports=lineInfo;